src/components/modal/Modal.scss

```diff
@@ -144,6 +144,10 @@
             padding: 0;
         }
 
+        .modal-content {
+            padding-bottom: 10 * $bdl-grid-unit;
+        }
```

Comment:
I'm not sure if this is avoidable without JS but this will add a padding even if there's no `.modal-actions` component. E.g. a modal that has content greater than the height but does not have action buttons will have extra bottom padding. Do we need to check with design if that's alright?

Reply:
Also should this be `padding-bottom: 15 * $bdl-grid-unit;`? I think the `.modal-actions` with the new padding is around 60px right?

Reply:
We can check with design, but this extra padding doesn't affect the modal display since the only case that content can scroll is the Move/Copy modal and then it's necessary. In any other case, the padding is not enough to make content scroll.

Reply:
The button bar height is 40px :D

Reply:
Given that the rest of the styles don't conform to the grid, I would suggest hardcoding the pixel values to stay consistent.

Reply:
@jstoffan Sounds good.

@msorianoBox We also have 10px padding above and below it I think we'll need to account for. Otherwise there will still be some content that's cut off.

<img width="360" alt="Screen Shot 2022-05-19 at 3 05 37 PM" src="https://user-images.githubusercontent.com/7311041/169412385-a2499f79-812e-4745-bed7-9aaf01e1cc05.png">